routes/backend.php

```diff
@@ -15,8 +15,8 @@ function () {
 );
 
 $router->group(['prefix' => 'wiki'], function () use ($router) {
-    // GET
     $router->get('getWikiForDomain', ['uses' => 'WikiController@getWikiForDomain']);
+    $router->patch('updateEntityImport', ['uses' => '\App\Http\Controllers\WikiEntityImportController@update']);
```

Comment:
is http `PATCH` used here because of semantics? not against it, just wondering because I'm not sure I encountered it in this repo before

Reply:
Yeah, mostly, you can read using GET, create using POST and update using PATCH.